src/shared/services/bolt/boltConnection.js

```diff
@@ -283,7 +283,7 @@ export const closeConnection = () => {
 }
 
 export const ensureConnection = (props, opts, onLostConnection) => {
-  const session = _drivers ? _drivers.getDirectDriver().session() : false
+  const session = _drivers ? _drivers.getRoutedDriver().session() : false
```

Comment:
@oskarhane whats the reason for this change?

Reply:
No reason :)
Removing that.